Automattic/wp-calypso
# Try Jetpack Cloud Sections in WordPress.com

*[ UPDATE: Added change info, test instructions, and screenshots. -- @mattgawarecki  ]*

#### Changes proposed in this Pull Request

* Exposes a Jetpack sidebar section via the feature flag `jetpack/features-section`.

Fixes `1179060693083348-as-1179411497904275`

#### Testing instructions

* Start Calypso in your local environment or on the Calypso live branch.
* Add `?flags=jetpack/features-section` to your page's URL.
* Observe that a Jetpack section appears in the sidebar.

#### Screenshots

<img width="301" alt="image" src="https://user-images.githubusercontent.com/670067/84080857-80dd3c00-a9a2-11ea-88cd-a8cbd01524e2.png">
